src/output.rs

```diff
@@ -83,31 +83,39 @@ impl OutputType {
             let replace_arguments_to_less = pager.source == PagerSource::EnvVarPager;
 
             if args.is_empty() || replace_arguments_to_less {
-                p.arg("--RAW-CONTROL-CHARS");
-                if single_screen_action == SingleScreenAction::Quit {
-                    p.arg("--quit-if-one-screen");
-                }
-
-                if wrapping_mode == WrappingMode::NoWrapping(true) {
-                    p.arg("--chop-long-lines");
-                }
-
                 // Passing '--no-init' fixes a bug with '--quit-if-one-screen' in older
                 // versions of 'less'. Unfortunately, it also breaks mouse-wheel support.
                 //
                 // See: http://www.greenwoodsoftware.com/less/news.530.html
                 //
                 // For newer versions (530 or 558 on Windows), we omit '--no-init' as it
                 // is not needed anymore.
-                match retrieve_less_version(&pager.bin) {
+                let less_version = retrieve_less_version(&pager.bin);
+                match less_version {
                     None => {
                         p.arg("--no-init");
```

Comment:
Do we want to pass this to BusyBox too? Isn't it better to not touch arguments at all if BuxyBox is detected? Right now the diff makes it a bit hard to quickly see if we behave the same when `less` is detected. Don't be afraid of introducing helper functions to improve structure and readability.

Reply:
Ok so there were two ways to go with this. BusBox doesn't support colored output, but it can filter out the escape sequences using the `-R` flag. So one can either turn off the color formatting in bat, or just pass a flag to have BusyBox ignore it. My thinking was that the latter was the less invasive to bat's codebase, but it's admittedly not going to be as efficient when using bat with busybox less (although probably not noticably so). My personal preference is for simpler code here, since BusyBox less being the only pager is admittedly an edge case.

As an aside, BusyBox doesn't support long options at all, so all of the other flags break it, which is what caused the bug report in the first place. But giving it a second look, it might also be handy to pass `-F` so that it quits if the output fits on one screen, and there is also an option to suppress the tilde characters it normally displays after the end of the output.

Anyway, that was my reasoning. I'll put more time in if the maintainers think it should be done differently. Or I can just add to the comments above this block to explain why we're passing only `-R` to BusyBox.

Reply:
It's a bit hard to evaluate beforehand what approach would work best, so if it is OK with you, we maybe can iterate a bit on this? I added another comment with a proposal on how we can maybe reuse code for both less and busybox.

The guiding light should be:
* minimize diff and only make "obviously correct" changes so that we don't accidentally introduce a regression with how we modify regular less arguments. I consider changing from "less -RF" to "less -FR" as unnecessary risk, to give a concrete example (that does not apply to your change, but I think you get the point).